cavy.js

```diff
@@ -55,6 +56,10 @@ program
     'Swap the index files and start the report server without first building the app'
   )
   .option('-d, --dev', 'Keep report server alive until manually killed')
+  .option(
+    '-bt, --boot-timeout <seconds>',
+    'Set how long the CLI should wait for the RN app to boot (defaults to 20 seconds)'
```

Comment:
How did you pick the default? Is this default enough time for our sample app to build in CircleCI?

Reply:
@jalada What do you think is a reasonable default for the boot timeout?
At the moment, the default is 20 seconds which is only reasonable for the tiny sample app.

Reply:
[CircleCI has a 10 min no-output timeout](https://support.circleci.com/hc/en-us/articles/360007188574-Build-has-hit-timeout-limit). So somewhere between 20 seconds and 10 minutes. How about 2 minutes?